pandas/core/reshape/tile.py

```diff
@@ -262,6 +264,9 @@ def qcut(x, q, labels=None, retbins=False, precision=3, duplicates='raise'):
         The precision at which to store and display the bins labels
     duplicates : {default 'raise', 'drop'}, optional
         If bin edges are not unique, raise ValueError or drop non-uniques.
+    bounded : bool, optional
+        Use the min/max of the distribution as the lower/upper bounds if True,
+        otherwise use -inf/inf. Ignored if dtype is datetime/timedelta.
```

Comment:
Move this below the version added, and add your own for `bounded` i.e.:

~~~
   .. versionadded:: 0.20.0
bounded: bool, optional
...
   .. versionadded:: 0.24.0
~~~